src/Biconomy.js

```diff
@@ -321,10 +321,13 @@ Biconomy.prototype.getForwardRequestAndMessageToSign = function (
         }
         _logMessage("API found");
         let params = methodInfo.params;
+        let typeString = "";
         let paramArray = [];
         for (let i = 0; i < params.length; i++) {
           paramArray.push(_getParamValue(params[i]));
+          typeString = typeString + params[i].type.toString()+",";
         }
+        typeString = typeString.substring(0,typeString.length - 1);
```

Comment:
What will this return if params.length is 0?

Reply:
this should be enough

if(params.length > 0) {
        typeString = typeString.substring(0,typeString.length - 1);
        }